### ISSUE TITLE
[ASK] I can't getting start in WSL

### ISSUE THREAD

**Issue by b856741** [recommenders-team/recommenders#1975]
### Description

when I ran `pip install recommenders[examples]` , it shows 
![q1](https://github.com/recommenders-team/recommenders/assets/22851308/78a29144-5c93-48ed-a35d-bb8f6fd60643)
 on the bottom.
when I emitted and ran the next command  `python -m ipykernel install --user --name <environment_name> --display-name <kernel_name>` in wsl , it shows: `/home/bcs109110/miniconda3/envs/surprise/bin/python: No module named ipykernel
`. when I tried to download ipykernel, it shows  
![q2](https://github.com/recommenders-team/recommenders/assets/22851308/b9611411-ddc7-4192-9582-59ad8ed37eb5)
in the end. 
Does anyone have suggestions ?

**Comment by devc007** [comment]
Hello, I'm new to this community. If it's alright, could you please explain what you're currently working on or trying to achieve? I'm eager to learn more and get involved.

**Comment by miguelgfierro** [comment]
Can you show the output of `python -c "import recommenders; print(recommenders.__version__)"`

**Comment by b856741** [comment]
hello, it shows:Traceback (most recent call last):
  File "<string>", line 1, in <module>
ModuleNotFoundError: No module named 'recommenders'

**Comment by miguelgfierro** [comment]
yeah, so it seems that recommenders was not installed properly. It's weird because I see recommenders within the installed libraries in the first screenshot. Maybe you are not using the right env. Try:
```
conda create -n recommenders python=3.9.16
conda activate recommenders
pip install recommenders[examples]
python -c "import recommenders; print(recommenders.__version__)"
```

**Comment by miguelgfierro** [comment]
this should be fixed with the new release 1.2.0
